samples/directx/d3d10_interop.cpp

```diff
@@ -135,7 +135,7 @@ class D3D10WinApp : public D3DSample
         if (!m_cap.read(m_frame_bgr))
             return -1;
 
-        cv::cvtColor(m_frame_bgr, m_frame_rgba, COLOR_BGR2RGBA);
+        cv::cvtColor(m_frame_bgr, m_frame_rgba, cv::COLOR_BGR2RGBA);
```

Comment:
This file doesn't have build problems.
"cv::" is not necessary in this file, see line 25: "using namespace cv;"

There is problem in `opengl_interop.cpp` only (with missing "using namespace cv;"), which violates existed guidelines for OpenCV samples.
+ WinRT samples.

So it is better to check/add "using namespace cv;" instead.